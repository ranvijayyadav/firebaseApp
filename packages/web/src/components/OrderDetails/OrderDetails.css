.order-card {
    box-shadow: 0px 14px 80px rgba(34, 35, 58, 0.2);
    border-radius: 15px;
}

.table-text{
    text-align: left;
}