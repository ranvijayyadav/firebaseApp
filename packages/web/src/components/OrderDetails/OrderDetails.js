import React, { useContext, Fragment } from "react";

import { UserContext } from "./../../context/UserContext";

const OrdersDetails = () => {
    const { order } = useContext(UserContext);
    return (
        <Fragment>
            {(JSON.stringify(order))}
        </Fragment>
    );
};

export default OrdersDetails;
