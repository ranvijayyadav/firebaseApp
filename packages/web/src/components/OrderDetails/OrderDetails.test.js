import React from 'react';
import { shallow } from 'enzyme';
import OrderDetails from './OrderDetails';

const orderMock = {
    order: {
        title: "Dont let me think , this is a long book title {looks good Edited}",
        bookingDate: 1575306000,
        customer: {
            phone: "0504565722",
            email: "bobTheLast@mail.com",
            name: "long name to check view"
        },
        address: {
            country: "united state of america",
            zip: "20000",
            street: "long street to check view",
            city: "long city name that's hard "
        }
    }
};


const UserContext = React.createContext({ order: orderMock });
let wrapped = shallow(
    <UserContext.Provider
        value={{
            order: orderMock,
        }}
    >
        <OrderDetails />
    </UserContext.Provider>

);
describe('Order Details', () => {
    it("Should render Order Details properly", () => {
        expect(wrapped).toMatchSnapshot();
    });
})