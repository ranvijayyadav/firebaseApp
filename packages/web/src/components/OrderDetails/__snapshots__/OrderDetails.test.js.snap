// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order Details Should render Order Details properly 1`] = `<OrdersDetails />`;
